.eslintrc.js

```diff
@@ -116,5 +116,13 @@ module.exports = {
             { lineEndings: 'unix' },
         ],
         'aws-toolkits/no-only-in-tests': 'error',
+        // The following will place an error on the `fs-extra` import since we do not want it to be used for browser compatibility reasons.
+        // "no-restricted-imports": [
+        //     "error",
+        //     {
+        //         "name": "fs-extra",
+        //         "message": "Avoid fs-extra, use FileSystemCommon. Notify the Toolkit team if your required functionality is not available."
+        //     }
+        // ],
```

Comment:
When do we plan to enable this? Are we still missing uses of `fs-extra`?

Reply:
I just put this here since we will eventually want it. There are still many uses of `fs-extra`.

I'm wondering if we can just enable even with uses of fs-extra, will this cause CI to fail?